.github/workflows/test.yml

```diff
@@ -62,6 +70,12 @@ jobs:
           . /home/firedrake/firedrake/bin/activate
           python3 -m pip install .[demos,optimisation]
 
+      - name: Get Muller et al 2022 plate reconstructions
+        run: |
+          s3cmd --config=$HOME/.s3cfg/config get s3://gadopt/github-actions/Muller_etal_2022_SE_1Ga_Opt_PlateMotionModel_v1.2.zip .
+          unzip Muller_etal_2022_SE_1Ga_Opt_PlateMotionModel_v1.2.zip -d demos/gplates_global
+        shell: bash
```

Comment:
```suggestion
```

This shouldn't be needed, right?

Reply:
Considering this is the only place I am having it, yes!